manage/deploying/production/status_check.rst

```diff
@@ -0,0 +1,38 @@
+============================
+Status check
+============================
```

Comment:
no it should fit the text length, so it grows and shrinks with the title text itself. At least rst linters complain about it :-)